partcad/src/partcad/project_factory_git.py

```
@@ -162,7 +164,10 @@
                                 branch_name = remote_head.reference.name
                                 short_branch_name = branch_name[branch_name.find("/") + 1 :]
                                 pc_logging.debug("Refreshing the GIT branch: %s" % short_branch_name)
-                                origin.pull(short_branch_name)
+                                with pc_tracer.start_as_current_span(
+                                    "*ProjectFactoryGit._clone_or_update_repo.{Repo.pull}"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)